### PR TITLE
operator mongodb-atlas-kubernetes (2.1.0)

### PR DESCRIPTION
**Pull request by josvazg**
Thanks for submitting your Operator. Please check the below list before you create your Pull Request.

### New Submissions

* [ ] Are you familiar with our [contribution guidelines](https://github.com/redhat-openshift-ecosystem/community-operators-prod/blob/main/docs/contributing-via-pr.md)?
* [ ] Are you familiar with our [operator pipeline](https://github.com/redhat-openshift-ecosystem/community-operators-prod/blob/main/docs/community-operator-pipeline.md)?
* [ ] Have you tested your Operator with all Custom Resource Definitions [packaging](https://github.com/redhat-openshift-ecosystem/community-operators-prod/blob/main/docs/packaging-operator.md)?
* [ ] Have you tested your Operator in all supported [installation modes](https://github.com/redhat-openshift-ecosystem/community-operators-prod/blob/main/docs/operator-ci-yaml.md##Operator-versioning)?
* [ ] Have you considered whether you want to use [semantic versioning order](https://github.com/operator-framework/community-operators/blob/master/docs/operator-ci-yaml.md#semver-mode)?
* [ ] Is your submission [signed](https://github.com/redhat-openshift-ecosystem/community-operators-prod/blob/main/docs/contributing-prerequisites.md#sign-your-work)?
* [ ] Is operator [icon](https://github.com/redhat-openshift-ecosystem/community-operators-prod/blob/main/docs/packaging-operator.md#operator-icon) set?

### Your submission should not

* [X] Add more than one operator bundle per PR
* [X] Modify any operator
* [X] Rename an operator
* [X] Modify any files outside the above mentioned folders
* [X] Contain more than one commit. **Please squash your commits.**

### Operator Description must contain (in order)

1. [X] Description of the managed Application and where to find more information
2. [X] Features and capabilities of your Operator and how to use it
3. [X] Any manual steps about potential pre-requisites for using your Operator

### Operator Metadata should contain

* [X] Human readable name and 1-liner description about your Operator
* [X] Valid [category name](https://github.com/redhat-openshift-ecosystem/community-operators-prod/blob/main/docs/packaging-operator.md#categories)<sup>1</sup>
* [X] One of the pre-defined [capability levels](https://github.com/operator-framework/operator-courier/blob/4d1a25d2c8d52f7de6297ec18d8afd6521236aa2/operatorcourier/validate.py#L556)<sup>2</sup>
* [X] Links to the maintainer, source code and documentation
* [X] Example templates for all Custom Resource Definitions intended to be used
* [X] A quadratic logo

Remember that you can preview your CSV [here](https://operatorhub.io/preview).

--

<sup>1</sup> If you feel your Operator does not fit any of the pre-defined categories, file an issue against this repo and explain your need

<sup>2</sup> For more information see [here](https://sdk.operatorframework.io/docs/overview/#operator-capability-level)

### CODE REVIEW

File: operators/mongodb-atlas-kubernetes/2.1.0/manifests/mongodb-atlas-kubernetes.clusterserviceversion.yaml
Comment (josvazg) [inline operators/mongodb-atlas-kubernetes/2.1.0/manifests/mongodb-atlas-kubernetes.clusterserviceversion.yaml:130]:
```suggestion
    containerImage: quay.io/mongodb/mongodb-atlas-kubernetes-operator:2.1.0
```

File: operators/mongodb-atlas-kubernetes/2.1.0/manifests/mongodb-atlas-kubernetes.clusterserviceversion.yaml
Comment (josvazg) [inline operators/mongodb-atlas-kubernetes/2.1.0/manifests/mongodb-atlas-kubernetes.clusterserviceversion.yaml:518]:
```suggestion
                    image: quay.io/mongodb/mongodb-atlas-kubernetes-operator:2.1.0
```

**Comment by josvazg** [comment]
cc @mporrato @Allda 
This one is also stuck after passing tests and we do not know what needs to be done to progress it forward.

**Comment by Allda** [comment]
@josvazg As stated in the previous comment this PR needs approval from at least one operator owner defined in the ci.yaml file. https://github.com/redhat-openshift-ecosystem/community-operators-prod/pull/4051#issuecomment-1948662529 
Once the PR got approved the pipeline will continue and get merged.

**Comment by josvazg** [comment]
> @josvazg As stated in the previous comment this PR needs approval from at least one operator owner defined in the ci.yaml file. [#4051 (comment)](https://github.com/redhat-openshift-ecosystem/community-operators-prod/pull/4051#issuecomment-1948662529) Once the PR got approved the pipeline will continue and get merged.

Thanks @Allda ! And sorry for the noise.

I make note of this for our release documentation, and will ask @helderjs or @igor-karpukhin to help me with this PR.

I am guessing the situation is similar for https://github.com/k8s-operatorhub/community-operators/pull/3904 ?

**Review by helderjs — APPROVED** [review]
lgtm

**Pull request merged**
